src/optimizer/plan_generator.py

```diff
@@ -95,18 +95,18 @@ def optimize(self, logical_plan: Operator):
     def build(self, logical_plan: Operator):
         # apply optimizations
 
-        return self.optimize(logical_plan)
+        self.optimize(logical_plan)
```

Comment:
@xzdandy @gaurav274 
By fixing this, some of plan generator tests can pass [here](https://github.com/georgia-tech-db/eva/pull/136/commits/c99a3094d58b0dc84595e03e78038e510d97fa64). Other test cases still fail because of missing implementation rule. Hence, the winner of each group is empty. Should I introduce more implementation rules for those missing logical operators?

Reply:
We miss limit and order by right? Do we miss any other implementation rules?

Reply:
We also need Project, Filter, and Orderby.

Reply:
Got. I think you can add them. They should be pretty straightforward. Or I can do that at a little late times.

Reply:
Sure, I can work on Project and Filter first.